Dockerfile

```diff
@@ -0,0 +1,33 @@
+#Another attempt at dockerizing frontend
+
+
+# Stage 1: Compile and Build angular codebase
+
+# Use official node image as the base image
+FROM node:latest as build
+
+# Set the working directory
+WORKDIR /app
+
+# Add the source code to app
+COPY ./p3-frontend .
+
+# Install all the dependencies
+#RUN cd ./p3-frontend
+RUN npm install
+
+# Generate the build of the application
+RUN npm run build
+
+
+# Stage 2: Serve app with nginx server
+
+# Use official nginx image as the base image
+FROM nginx:latest
```

Comment:
we don't want to be blindsided by breaking changes here either, but security updates are good to get automatically.
check out [nginx's versioning scheme](https://www.nginx.com/blog/nginx-1-18-1-19-released/) and decide for yourself what image tag to use.

Reply:
removed docker files from this feature branch.

Reply:
Set versioning per your suggestion.